EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -59,9 +59,27 @@
 ctppsDiamondRawToDigi.rawDataTag = cms.InputTag("rawDataCollector")
 
 
+# pixels
+ctppsPixelDAQMappingESSourceXML = cms.ESSource("CTPPSPixelDAQMappingESSourceXML",
+  verbosity = cms.untracked.uint32(0),
+  subSystem= cms.untracked.string("RPix"),
+  configuration = cms.VPSet(
+  # example configuration block:
+  cms.PSet(
+    validityRange = cms.EventRange("1:min - 999999999:max"),
+    mappingFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/rpix_mapping_220_far.xml"),
+    maskFileNames = cms.vstring("CondFormats/CTPPSReadoutObjects/xml/rpix_channel_mask_220_far.xml")
+    )
+  )
+)
+
+from EventFilter.CTPPSRawToDigi.ctppsPixelRawToDigi_cfi import ctppsPixelDigis
+ctppsPixelDigis.rawDataTag = cms.InputTag("rawDataCollector")
```

Comment:
does ``` rawDataTag ```  exist for this module?
Why is it not in the default cfi in ctppsPixelRawToDigi_cfi.py ?